events/tests/test_event_get.py

```diff
@@ -603,6 +603,37 @@ def test_admin_user_filter(api_client, event, event2, user):
     get_list_and_assert_events("admin_user=true", [event], api_client)
 
 
+@pytest.mark.django_db
+def test_registration_admin_user_filter(
+    event, event2, event3, organization3, user, user_api_client
+):
+    event3.publisher = organization3
+    event3.save()
+    event2.publisher.registration_admin_users.add(user)
+    get_list_and_assert_events(
+        "registration_admin_user=false", [event, event2, event3], user_api_client
+    )
+    event2.publisher.registration_admin_users.add(user)
+    get_list_and_assert_events(
+        "registration_admin_user=true", [event, event2], user_api_client
+    )
+
+
+@pytest.mark.django_db
+def test_cannot_use_registration_admin_user_and_admin_user_filters_simultaneously(
+    user_api_client,
+):
+    url = "%s?%s" % (
+        reverse("event-list"),
+        "admin_user=true&registration_admin_user=true",
+    )
+    response = response = api_client.get(url, format="json")
```

Comment:
```suggestion
    response = api_client.get(url, format="json")
```

Reply:
Removed the extra `response =` and added check for the `status_code`